cpp/data/installer/tuefind.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE vufind.user ADD ub_subscribed_to_newsletter BOOL NOT NULL DEFAULT FALSE;
```

Comment:
Contents of this file should be moved to ubtue/tuefind repository (module/TueFind/sql/mysql.sql
(this file is already used by the installer)

Reply:
Also, prefix should not be "ub_" but "tuefind_".